src/playback/details.jsx

```diff
@@ -61,17 +61,22 @@ class PlaybackDetails extends React.Component {
                     encoding,
                     size,
                   })
-                  .map(({ speed, filter: testFilter }) => ({
-                    label: `${speed}x`,
-                    repo: browserDetails.repo,
-                    filter: {
-                      and: [
-                        platformDetails.filter,
-                        browserDetails.filter,
-                        testFilter,
-                      ],
-                    },
-                  }))
+                  .map(test => {
+                    const playbackSuite = selectFrom(PLAYBACK_SUITES).where({ variant: test.variant, browser }).first();
+                    const suiteFilter = { eq: { suite: playbackSuite.suite } };
```

Comment:
instead of 

                    const playbackSuite = selectFrom(PLAYBACK_SUITES).where({ variant: test.variant, browser }).first();
                    const suiteFilter = { eq: { suite: playbackSuite.suite } };

try

                    const suite = selectFrom(PLAYBACK_SUITES)
                        .where({ variant: test.variant, browser })
                        .first()
                        .suite;
                    const suiteFilter = { eq: { suite } };

which is minor, and not worth changing